scene/gui/tab_container.cpp

```diff
@@ -165,21 +186,22 @@ void TabContainer::_notification(int p_what) {
 				return;
 			}
 
-			int header_height = _get_top_margin();
+			int header_height = _get_tab_height();
+			int header_voffset = (tabs_position == POSITION_BOTTOM) * (size.height - header_height);
 
 			// Draw background for the tabbar.
-			theme_cache.tabbar_style->draw(canvas, Rect2(0, 0, size.width, header_height));
+			theme_cache.tabbar_style->draw(canvas, Rect2(0, header_voffset, size.width, header_height));
 			// Draw the background for the tab's content.
-			theme_cache.panel_style->draw(canvas, Rect2(0, header_height, size.width, size.height - header_height));
+			theme_cache.panel_style->draw(canvas, Rect2(0, (tabs_position == POSITION_TOP) * header_height, size.width, size.height - header_height));
```

Comment:
Booleans in calculations are ok, but it's better if they are explicit IMO.
```suggestion
			theme_cache.panel_style->draw(canvas, Rect2(0, int(tabs_position == POSITION_TOP) * header_height, size.width, size.height - header_height));
```